ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -125,7 +125,7 @@ def verify_image_versions(old_images, upgrade_version, version_before_upgrade):
     verify_pods_upgraded(old_images, selector=constants.MDS_APP_LABEL, count=2)
     if config.ENV_DATA.get("platform") in constants.ON_PREM_PLATFORMS:
         rgw_count = get_rgw_count(
-            float(upgrade_version), True, float(version_before_upgrade)
+            upgrade_version.base_version, True, version_before_upgrade
```

Comment:
Is base version float or str? As you accepting those in function now.

Reply:
The function accepted both already I just updated the docstring to reflect that. `base_version` is a string:

```shell
>>> ver = parse_version('4.5')
>>> type(ver)
<class 'pkg_resources.extern.packaging.version.Version'>
>>> type(ver.base_version)
<class 'str'>
>>> ver
<Version('4.5')>
>>> ver.base_version
'4.5'
```